fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkexport/ChunkWriter.java

```diff
@@ -139,30 +142,46 @@ private void pushFhirJsons2Cos(InputStream in, int dataLength) throws Exception
 
             String itemName;
             if (cosBucketPathPrefix != null && cosBucketPathPrefix.trim().length() > 0) {
-                itemName = cosBucketPathPrefix + "/" + fhirResourceType + "_" + chunkData.getPartNum() + ".ndjson";
+                itemName = cosBucketPathPrefix + "/" + ResourceTypes.get(chunkData.getIndexOfCurrentResourceType())
```

Comment:
just a thought here... 

maybe it's worth externalizing from ChunkWriter the path prefixes and patterns.  
If you externalize, it could have a per client or per bucket implementation? 

this is a design discussion, not meant to re-program/fix error (I don't see any here).

Reply:
this cosBucketPathPrefix is passed from the client side, and it's optional for client. if not set, then job# will be used as prefix, otherwise, the passed-in cosBucketPathPrefix will be used.